app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if not command:
+        return
+    command_list = command.split()
+    mv, file_name, new_file_name = command_list
+    if mv == "mv" and len(command_list) == 3:
```

Comment:
first check that you can unpack, that is, whether the command_list contains three elements, and only then unpack